packages/teuchos/comm/test/StackedTimer/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+TRIBITS_INCLUDE_DIRECTORIES(REQUIRED_DURING_INSTALLATION_TESTING ${CMAKE_CURRENT_SOURCE_DIR})
+
+TRIBITS_ADD_EXECUTABLE_AND_TEST(
```

Comment:
This is not actually a test.  All it does is run some code.  You could gut the 'stacked_timer' executable and the "test" would still pass.